user/conditions-v1.md

```diff
@@ -0,0 +1,320 @@
+---
+title: Conditions
+layout: en
+---
+
+> This page documents the behavior of conditions `v1`, which is currently
+> available using `conditions: v1` in your `.travis.yml` file. This new
```

Comment:
This whole note isn't very easy to parse. What do you think of something like 

> This page documents conditions v1, which you can use by setting `conditions: v1` in your `.travis.yml`. v1  replaces [`v0`](/user/conditions-v0) which is being deprecated.

Reply:
sounds good! i've kept the link to the `v0` docs though?

> This page documents conditions `v1`, which you can use by setting conditions:
> `v1` in your `.travis.yml`. `v1` replaces `v0` which is being deprecated.
> The documentation for `v0` can be found [here](/user/conditions-v0).

Reply:
LGTM